collector/service.go

```diff
@@ -14,15 +16,30 @@ func init() {
 	Factories["service"] = NewserviceCollector
 }
 
+var (
+	serviceWhereClause = flag.String("collector.service.services-where", "", "WQL 'where' clause to use in WMI metrics query. Limits the response to the services you specify and reduces the size of the response.")
+)
+
 // A serviceCollector is a Prometheus collector for WMI Win32_Service metrics
 type serviceCollector struct {
 	State     *prometheus.Desc
 	StartMode *prometheus.Desc
+
+	queryWhereClause string
 }
 
 // NewserviceCollector ...
 func NewserviceCollector() (Collector, error) {
 	const subsystem = "service"
+
+	var wc bytes.Buffer
+	if *serviceWhereClause != "" {
+		wc.WriteString("WHERE ")
+		wc.WriteString(*serviceWhereClause)
+	} else {
+		log.Println("warning: No where-clause specified for process collector. This will generate a very large number of metrics!")
```

Comment:
I don't think we need this warning for the service collector, there won't be that many metrics.